backend/typescript/config/config.json

```diff
@@ -0,0 +1,38 @@
+{
+  "development": {
+    "use_env_variable": "DATABASE_URL",
+    "dialect": "postgres",
+    "protocol": "postgres",
+    "ssl": true,
+    "dialectOptions": {
+      "ssl": {
+        "require": true,
+        "rejectUnauthorized": false
+      }
+    }
+  },
+  "staging": {
```

Comment:
this corresponds to the `NODE_ENV` environment variable - we'll need to make sure this is set accordingly in heroku